uploader.json

```diff
@@ -2810,6 +2810,16 @@
                 "reserved_port/range"
             ],
             "symbolic_name": "sap_host_profile"
+        },
+        {
+            "file": "/etc/ironic/ironic.conf",
+            "pattern": [],
+            "symbolic_name": "ironic_conf"
+        },
+        {
+            "file": "/var/lib/config-data/puppet-generated/ironic/etc/ironic/ironic.conf",
+            "pattern": [],
+            "symbolic_name": "ironic_conf"
```

Comment:
@smalleni please move these entries so that the symbolic name is sorted alphabetically in the list of files.